packages/mdc-chips/chip/test/foundation.test.ts

```diff
@@ -0,0 +1,703 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+
+import {verifyDefaultAdapter} from '../../../../testing/helpers/foundation';
+import {setUpFoundationTest, setUpMdcTestEnvironment} from '../../../../testing/helpers/setup';
```

Comment:
Does Copybara transform work for 4 level depth relative path?

Reply:
It doesn't work - it tries to change `testing` to `mdc-testing`

Reply:
Did you mean it does work? I assume it worked when you exported the CL :)

Reply:
It doesn't - I had to manually copy and paste and change the import paths for now

Reply:
Export also doesn't work yet for all the other migrated components (i.e. checkbox), I can run a test PR to demonstrate

Reply:
See here: https://github.com/material-components/material-components-web/pull/5376/files

Reply:
Ah ok. Seems like Copybara config file needs to be updated to transform imports in tests. Exporting to PR should just work without any manually changes.

I'll update the Copybara transformation.

Reply:
Thanks! Sorry about the misleading description